### ISSUE TITLE
[Bug] - Sampler Node Crashes When Switching Position to Relative Units

### ISSUE THREAD

**Issue by Synthoid** [Ttanasart-pt/Pixel-Composer#15]
Attempting to change the `Position` value of a `Sampler` node to use relative units instead of pixels throws the following error then crashes the application:
```
___________________________________________
############################################################################################
ERROR in
action number 1
of Draw Event
for object o_main:

I32 argument is unset############################################################################################
gml_Script_anon_Node_Processor_gml_GlobalScript_node_processor_605126_Node_Processor_gml_GlobalScript_node_processor (line 29)
gml_Script_anon_Node_Sampler_gml_GlobalScript_node_color_sampler_364399_Node_Sampler_gml_GlobalScript_node_color_sampler (line 8)
gml_Script_anon_nodeValueUnit_gml_GlobalScript_node_value_4127657_nodeValueUnit_gml_GlobalScript_node_value (line 202)
gml_Script_anon_NodeValue_gml_GlobalScript_node_value_23070707_NodeValue_gml_GlobalScript_node_value (line 857)
gml_Script_anon_nodeValueUnit_gml_GlobalScript_node_value_3289658_nodeValueUnit_gml_GlobalScript_node_value (line 170)
gml_Script_anon_buttonClass_gml_GlobalScript_button_619727_buttonClass_gml_GlobalScript_button (line 40)
gml_Script_anon_nodeValueUnit_gml_GlobalScript_node_value_3467654_nodeValueUnit_gml_GlobalScript_node_value (line 179)
gml_Script_anon_vectorBox_gml_GlobalScript_vectorBox_912614_vectorBox_gml_GlobalScript_vectorBox (line 50)
gml_Script_anon_Panel_Inspector_gml_GlobalScript_panel_inspector_183580_Panel_Inspector_gml_GlobalScript_panel_inspector (line 276)
gml_Script_anon_scrollPane_gml_GlobalScript_scrollPane_768582_scrollPane_gml_GlobalScript_scrollPane (line 45)
gml_Script_drawContent_Panel_Inspector_gml_GlobalScript_panel_inspector (line 496)
gml_Script_draw_PanelContent_gml_GlobalScript_panel_data (line 456)
gml_Script_drawPanel_Panel_gml_GlobalScript_panel_data (line 370)
gml_Script_draw_Panel_gml_GlobalScript_panel_data (line 306)
gml_Script_draw_Panel_gml_GlobalScript_panel_data (line 322)
gml_Script_draw_Panel_gml_GlobalScript_panel_data (line 322)
gml_Object_o_main_Draw_64 (line 4)
```
![image](https://user-images.githubusercontent.com/6645950/209891005-fd03a451-8109-4b33-8560-32b6602ee976.png)

This was seen on `v 1.0.7`

**Comment by Ttanasart-pt** [comment]
Hmmm... I can't replicate this error on 1.0.8, maybe some modification I made fixed it, I'll need you to check again once I update it, thank you for reporting.

**Comment by Synthoid** [comment]
This crash no longer seems to be happening as of `1.0.9`. However, I am unable to adjust the position value properly once it has been switched to fraction units. Any decimal number I input (ie `0.5`, `1.7`, etc) will be rounded to the nearest whole number. The pixel value is converted into the correct fraction unit, but attempting to adjust fraction unit values directly causes this issue.

**Comment by Ttanasart-pt** [comment]
Thanks for reporting. This one should be fix in the next version.

**Comment by Synthoid** [comment]
Seemed to be fixed in `1.1.0`. Nice work!
